server/index.js

```diff
@@ -1,11 +1,5 @@
-const express = require('express');
-const db = require('./db/models/index.js');
 
 const PORT = process.env.PORT || 3004;
+const app = require('./app.js');
 
-const app = express();
-app.use(express.static('public'));
-app.get('/app.bundle.js', (req, res) => {
-  res.sendFile('public/app.bundle.js', { root: '../' });
-});
```

Comment:
It's not a problem at all, I'm curious why you split the previous section into a separate file?

Reply:
I want to keep my routes separately so i can easily import and test it without messing my app server